 odk/docs/cpp/Doxyfile |    1 +
 odk/docs/idl/Doxyfile |    1 +
 2 files changed, 2 insertions(+)

New commits:
commit df4877382c19042e267d6a8ed7b9a93f75509bdd
Author: David Tardon <dtardon@redhat.com>
Date:   Thu May 15 13:41:16 2014 +0200

    fix for doxygen 1.8.7
    
    Earlier versions apparently used HTML as default if no output format was
    specified. 1.8.7 no longer does that.
    
    Change-Id: Ia8f908f54b344530e1b50e8800c032f782fb36e9
    (cherry picked from commit 2e9a77aa2d8e83f6296cd9204759d18b0ec5f0b5)
    Reviewed-on: https://gerrit.libreoffice.org/9362
    Reviewed-by: Caolán McNamara <caolanm@redhat.com>
    Tested-by: Caolán McNamara <caolanm@redhat.com>

diff --git a/odk/docs/cpp/Doxyfile b/odk/docs/cpp/Doxyfile
index 26c8893..c34915a 100644
--- a/odk/docs/cpp/Doxyfile
+++ b/odk/docs/cpp/Doxyfile
@@ -24,6 +24,7 @@ WARN_FORMAT = "$file:$line: $text"
 INPUT = %
 RECURSIVE = YES
 
+GENERATE_HTML = YES
 HTML_OUTPUT = .
 
 GENERATE_LATEX = NO
diff --git a/odk/docs/idl/Doxyfile b/odk/docs/idl/Doxyfile
index 6eead54..c065b33 100644
--- a/odk/docs/idl/Doxyfile
+++ b/odk/docs/idl/Doxyfile
@@ -13,6 +13,7 @@ STRIP_FROM_PATH = %
 INPUT = %
 RECURSIVE = YES
 
+GENERATE_HTML = YES
 HTML_OUTPUT = .
 GENERATE_LATEX = NO
 
